po/pt_BR.po

```diff
@@ -5951,3 +5951,15 @@ msgid ""
 "\n"
 "%s"
 msgstr "Erro ao abrir o navegador web:\n\n%s"
+
+#: picard/ui/theme.py:39
+msgid "Light"
+msgstr "Claro"
```

Comment:
We use transifex, so I don't think those changes should be included.
We'll update `picard.pot` after the merge.

Reply:
Sure, I was just testing if it worked correctly (failed miserably in using the translations in the local build). I'm going to squash and fix the commits. :)

Reply:
I don't really remember. Should I update something with the new entries to be translated?

Reply:
To test translations locally, check https://github.com/metabrainz/picard/blob/master/INSTALL.md#running-from-the-source-tree
Basically, after updating one of `locale/*/*.po`, just run `python setup.py build_locales -i` so it compiles po files.

Reply:
What we'll do after the patch is merged:
- run `python setup.py regen_pot_file` and commit changes
- transifex will pick up this file within 6 hours (4 times a day)
- new translatable strings will be available on transifex
- before next release, we'll run `python setup.py pull_translations` to resync repo `.po` files with transifex